src/components/forms/YamlForm.tsx

```diff
@@ -46,7 +46,10 @@ const YamlForm: FC<Props> = ({
   return (
     <>
       {children}
-      <div ref={containerRef} className="code-editor-wrapper">
+      <div
+        ref={containerRef}
+        className="code-editor-wrapper restrict-editor-width"
```

Comment:
Why introduce a new class? We can use `code-editor-wrapper` to limit the editor width, no?

Reply:
the `code-editor-wrapper` class is nested in the `cloud-init-config` class which isn't applicable for all forms. I will create a new selector with the same class but not nested instead.